ansible/setup.yml

```diff
@@ -0,0 +1,80 @@
+#################################################
+# Initial Server Setup for Radicle Components
+#################################################
+---
+- hosts: all
+  vars:
+      ansible_python_interpreter: /usr/bin/python3
+      ansible_ssh_common_args: '-o StrictHostKeyChecking=no'
+      #RAD_SUBGRAPH: https://...
+      #RAD_RPC_URL: wss://...
+      #RAD_ORG: 0x...
+
+  tasks: 
+    - name: Create Directories
+      file:
+        path: "{{ item }}"
+        state: directory
+      with_items:
+      - /app/radicle/root
+
+    - name: Copy Org Identity File
+      copy:
+        src: ../keys/{{ RAD_ORG }}.key
+        dest: /app/radicle/identity
+
+    - name: Add RAD Environment Variables
+      lineinfile: dest=/root/.profile line="{{ item }}" insertafter='EOF' regexp="{{ item }}" state=present
+      with_items:
+      - 'export RAD_SUBGRAPH={{ RAD_SUBGRAPH }}'
+      - 'export RAD_ORG={{ RAD_ORG }}'
+      - 'export RAD_RPC_URL={{ RAD_RPC_URL }}'
+
+    - name: Install pip3
+      apt: 
+        name: python3-pip
+        state: present
+        update_cache: yes
+
+    - name: Install docker-py
+      pip:
+        name: docker-py
+
+    - name: Pull Docker Images
+      docker_image:
+        name: "{{ item }}"
+      with_items:
+        - gcr.io/radicle-services/org-node:latest
+        - gcr.io/radicle-services/http-api:latest
+        - gcr.io/radicle-services/git-server:latest
+  
+    - name: Start org-node
+      docker_container:
+        name: org-node
+        image: gcr.io/radicle-services/org-node:latest
+        volumes:
+          - /app
+        ports:
+          - "8776:8776/udp"
+        command: "--subgraph {{ RAD_SUBGRAPH }} --orgs {{ RAD_ORG }} --rpc-url {{ RAD_RPC_URL }}"
+
+    - name: Start http-api
+      docker_container:
+        name: http-api
+        image: gcr.io/radicle-services/http-api:latest
+        volumes:
+          - /app
+        ports:
+          - "8777:8777"
+        #command: "--tls-cert /app/radicle/fullchain.pem --tls-key /app/radicle/privkey.pem"
+        restart_policy: always
+
+    - name: Start git-server
+      docker_container:
+        name: git-server
+        image: gcr.io/radicle-services/git-server:latest
+        volumes:
+          - /app
+        ports:
+          - "8778:8778"
+        #command: "--tls-cert /app/radicle/fullchain.pem --tls-key /app/radicle/privkey.pem"    
```

Comment:
Should we handle HTTPS with caddy, like we currently do?

Reply:
Yes, this part needs DNS records, so we can generate certs using something like Certbot/Let's Encrypt. They're left unimplemented for now. As per caddy specifically, I've never had to use it before.

Reply:
Caddy handles fetching and renewing certs and everything :)

Reply:
Does it do the initial setup as well, meaning `--tls-cert` is placeholder or needs to be created beforehand?

Reply:
It does everything yeah, ie. there's no need to use `--tls-cert` at all if you use caddy in front.